packages/postcss/mixin.core.js

```diff
@@ -13,17 +9,46 @@ const getAssetPath = (...args) => trimSlashes(join(...args));
 
 const cssLoaderLocalOptions = {
   modules: {
+    auto: () => true,
     localIdentName: '[folder]-[name]-[local]-[hash:8]',
+    exportLocalsConvention: 'camelCase',
   },
-  localsConvention: 'camelCase',
+  esModule: false,
```

Comment:
`esModule` is `true` by default for all the CSS-related Webpack loaders & plugins. Explicitly setting it to `false` throughout this file, to keep the status quo.

Reply:
I like the idea of changing its value in the next major release.

Reply:
Added another commit, that breakingly changes the value. We can omit that commit, when cherry-picking.